doc/source/index.rst

```diff
@@ -69,6 +69,36 @@ Operator Documentation
 
     installing_server
     installing_daemons
+    man/rucio-abacus-account
```

Comment:
That seems a bit spammy? 😄 I think we should put this in a separate section and not the main index.

Reply:
yes, it was to get them generated for now.